CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+### Added
+- Template Builder: Access Page
```

Comment:
@fraserclark could you add the ticket number alongside your entry, as well as linking to it from your PR's description section?